src/style.css

```diff
@@ -0,0 +1,124 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+
+  --main-font: rgba(6, 11, 53, 1);
+  --additional-font: rgba(97, 96, 112, 1);
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
```

Comment:
```suggestion
```
[STYLES] - Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.